pcsx2/GS.h

```diff
@@ -217,14 +217,14 @@ struct GSRegSIGBLID
 #define GSIMR		((u32&)*(PS2MEM_GS+0x1010))
 #define GSSIGLBLID	((GSRegSIGBLID&)*(PS2MEM_GS+0x1080))
 
-enum GS_RegionMode
+enum GS_VideoMode
```

Comment:
Either you use c++11 enum class or you use a prefix on the enum as a pseudo scope.